.github/workflows/ci.yml

```diff
@@ -58,18 +58,10 @@ jobs:
         stack-setup-ghc: true
         stack-no-global: true
 
-    # For some reason, installing happy from lts-9.0 fails unless some other
-    # version of happy is already in the PATH. So let's install a version of
-    # happy from an lts which does not have this problem.
-    - name: Install happy
+    - name: Build oldest
       if: matrix.stack_yaml == 'oldest-supported-lts.yaml'
```

Comment:
There is no longer any difference between the `Build oldest ` and `Build` steps, so let's combine them into a single step with no `if`.

Reply:
Sure